### DISCUSSION TITLE
wyłączenie walidacji parametrów produktu, jeśli oferta jest powiązana z katalogiem

### DISCUSSION THREAD

**Discussion by PhotoSoft** [allegro/allegro-api#6258] [Suggestion - verification]
Notorycznie zmieniacie parametry produktów w katalogu.

Co doprowadza do tego, że obecnie, jeśli sprzedawca chce zaktualizować lub wystawić ponownie ofertę z danym produktem z zewnętrznej aplikacji to dostanie błąd dotyczący nieprawidłowych parametrów. Dlatego, że każda (raczej) aplikacja posiada zapisane parametry w bazie, które przez to stają się co chwilę nieaktualne.

W związku z tym sprzedawca najpierw musi pobrać do bazy aktualne parametry produktu i dopiero może opublikować ofertę. Jest to strata czasu.

**Więc skoro produkt ma stałe parametry, których i tak nie można edytować to nie powinny one być w ogóle brane pod uwagę podczas walidacji oferty.**

**Comment by MaciejFrackowiak** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.
